generation/utils/downloadEspresso.js

```diff
@@ -0,0 +1,17 @@
+// Gets a fully qualified class path of espresso and downloads it in a temp dir
+const os = require("os");
+const downloadFileSync = require("download-file-sync");
+const fs = require("fs");
+
+module.exports = function downloadEspresso(fullyQualifiedClass) {
+	const tmpDir = os.tmpdir();
+	const path = fullyQualifiedClass.replace(/\./g, "/");
+	const fileContent = downloadFileSync(
```

Comment:
I suggest you use concurrency here, this may take long time downloading files individually. I know there are a few download managers on npm that do exactly that.

Reply:
Isn

Reply:
Isn't that a premature optimization? I mean this code is only executed while generating code 🤷‍♂️

Reply:
https://www.npmjs.com/package/download

Reply:
I gave it a try, but it made the code unnecessarily complicated to read, while I don't see this big speedup opportunities (we are talking about maybe 10 file in total, I guess). I would like to leave it this simple, but feel free to add the download package in 👍

Reply:
cool, had no idea were talking about 10 files

Reply:
recounted, it's two 😆